packages/fxa-settings/src/pages/Signin/SigninRecoveryChoice/container.test.tsx

```diff
@@ -0,0 +1,169 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as ModelsModule from '../../../models';
+import * as ReachRouterModule from '@reach/router';
+import * as CacheModule from '../../../lib/cache';
+import * as SigninRecoveryChoiceModule from './index';
+
+import { waitFor } from '@testing-library/react';
+import { LocationProvider } from '@reach/router';
+import { renderWithLocalizationProvider } from 'fxa-react/lib/test-utils/localizationProvider';
+import { MOCK_STORED_ACCOUNT, mockLoadingSpinnerModule } from '../../mocks';
+import { mockSigninLocationState } from '../mocks';
+import SigninRecoveryChoiceContainer from './container';
+import AuthClient from 'fxa-auth-client/lib/client';
+
+jest.mock('../../../models', () => {
+  return {
+    ...jest.requireActual('../../../models'),
+    useAuthClient: jest.fn(),
+  };
+});
+
+const mockAuthClient = new AuthClient('http://localhost:9000', {
+  keyStretchVersion: 2,
+});
+
+function mockModelsModule({
+  mockGetRecoveryCodesExist = jest.fn().mockResolvedValue({
+    hasBackupCodes: true,
+    count: 3,
+  }),
+  mockRecoveryPhoneGet = jest.fn().mockResolvedValue({
+    exists: true,
+    phoneNumber: '7890',
```

Comment:
Right now this will return `MOCK_MASKED_PHONE_NUMBER` FWIW, and not just the last 4, if the session isn't verified. I need to fill out details for FXA-11044.

Reply:
Thank you, I'll update to test that the masked number is handled properly!